backend/clickhouse/logs.go

```diff
@@ -294,16 +294,16 @@ func splitQuery(query string) []string {
 	inquote := false
 	i := 0
 	for j, c := range query {
-		if c == '\'' {
+		if c == '"' {
 			inquote = !inquote
 		} else if c == ' ' && !inquote {
 			result = append(result, unquoteAndTrim(query[i:j]))
-			i = j + i
```

Comment:
This was the bug that was causing #4359 because `i` would end up being too big when we tried to capture the rest of the string on line 304 below (`query[i:]`).